src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -60,9 +60,14 @@
                                          message-list)])}
                  {:header "Workflow ID" :starting-width 300}]
        :filters [{:text "All" :pred (constantly true)}
-                 {:text "Succeeded" :pred #(contains? moncommon/wf-success-statuses (% "status"))}
-                 {:text "Running" :pred #(contains? moncommon/wf-running-statuses (% "status"))}
-                 {:text "Failed" :pred #(contains? moncommon/wf-failure-statuses (% "status"))}]
+                 {:text "Queued" :pred #(= "Queued" (% "status"))}
+                 {:text "Launching" :pred #(= "Launching" (% "status"))}
+                 {:text "Submitted" :pred #(= "Submitted" (% "status"))}
+                 {:text "Running" :pred #(= "Running" (% "status"))}
+                 {:text "Aborting" :pred #(= "Aborting" (% "status"))}
+                 {:text "Succeeded" :pred #(= "Succeeded" (% "status"))}
+                 {:text "Failed" :pred #(= "Failed" (% "status"))}
+                 {:text "Aborted" :pred #(= "Aborted" (% "status"))}]
```

Comment:
This is a little repeaty, it would be nice to make a function to create the filter items, which could then be mapped over the list of statuses.  Also, is "unknown" still a status in Rawls?